### DISCUSSION TITLE
Local videos not accessible

### DISCUSSION THREAD

**Discussion by hdt25** [namidaco/namida#287] [General]
I scanned the local videos, but it's not showing while playing music.
There is an option to set YouTube video link to individual song.
But there is no way to set local video like this.

**Comment by MSOB7YY** [comment]
setting youtube link works by modifying track `comment` metadata tag.
the way local videos linked are a bit different, since they are local and their path could be changed anytime, so the only way is to keep their name the same as the track, and make sure that the folder is inside indexer folder.

check the https://github.com/namidaco/namida#video-integration section to know how matching is done
